apps/web/src/components/Onboarding/SetupPassword/SetupPassword.tsx

```diff
@@ -99,16 +134,24 @@ export const SetupPassword = ({ mode }: SetupPasswordProps) => {
             curve,
             isVerified: !isNewMnemonic,
           });
+
+          if (isNewMnemonic) {
+            dispatch(accountsActions.setPassword(password));
+          }
+
           break;
         }
+        case "verification": {
+          const mnemonic = await getDecryptedMnemonic(currentAccount, password);
+
+          return openWith(<ImportantNoticeModal seedPhrase={mnemonic} />, { size: "xl" });
```

Comment:
nit: for consistency, maybe it's better to use just 1 thing - either seedphrase or mnemonic